client/my-sites/theme/main.jsx

```diff
@@ -795,9 +795,10 @@ class ThemeSheet extends Component {
 
 		const launchPricing = () => window.open( plansUrl, '_blank' );
 
-		const { canonicalUrl, description, name: themeName } = this.props;
+		const { canonicalUrl, description, name: themeName, seo_title, seo_description } = this.props;
+
 		const title =
-			themeName &&
+			( seo_title || themeName ) &&
 			translate( '%(themeName)s Theme', {
 				args: { themeName },
 			} );
```

Comment:
```suggestion
			seo_title ? seo_title : translate( '%(themeName)s Theme', {
			args: { themeName },
		} );
```

I had a look, and the `seo_title` is defined from the API, it's just that in the code above we're showing the translated `%(themeName)s Theme` regardless. We're really asking `title` to be the translated generated title if either `seo_title` OR `themeName` is defined. 

We really want to ask `title`to be `seo_title` if `seo_title` is defined (i.e. returned from the API), and if not fallback to the `translate` function. 

If you do my change above, you can then run `yarn build-server && yarn start` and you'll see this title being set on the first page load via SSR. 

You should probably do a commit from your local Calypso, and not use the "apply selection" here, since It's a bit wonky when there's removed lines inside the suggestion, and the indents get a bit funny.

Reply:
You can also do, I guess, but it's a bit less clear IMO

```
seo_title ?? translate( '%(themeName)s Theme', {
			args: { themeName },
		} );
```